src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,55 @@
+import classnames from 'classnames';
+import { px } from '../../utils';
+import CloseIcon from '../../assets/close.svg?react';
+import Button from '../Button/Button';
+
+export interface ModalProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * Boolean that will determine if the modal is open
+   */
+  open?: boolean;
+  /**
+   * Function to close the modal
+   */
+  onClose?: () => void;
+}
+
+const onClosePlaceholder = () => {
+  console.log('On close clicked');
```

Comment:
I'd rather this be a noop instead of a local function.  I think we have a function like this defined in a shared utils file already @davidicus